shailja's work/Palindrome_program.py

```diff
@@ -0,0 +1,10 @@
+def pal_string(p):
+    return p == p[::-1]
```

Comment:
Do you think, that will eliminate all other special characters except alphanumerical value?